packages/web-app-files/src/helpers/resources.ts

```diff
@@ -359,7 +363,7 @@ export function buildSharedResource(share, incomingShares = false, allowSharePer
         shareType: ShareTypes.user.value
       }
     ]
-
+    resource.sharedWith = share.sharedWith
```

Comment:
Tests are failing because apparently the `shareWith` field is sometimes undefined. Please change this line as follows to get the tests hopefully passing again:
```suggestion
    resource.sharedWith = share.sharedWith || []
```